src/lib/isIdentityCard.js

```diff
@@ -130,15 +130,15 @@ const validators = {
   },
   IR: (str) => {
     if (!str.match(/^\d{10}$/)) return false;
-    str = (`0000${str}`).substr(str.length - 6);
+    str = (`0000${str}`).slice(str.length - 6);
 
-    if (parseInt(str.substr(3, 6), 10) === 0) return false;
+    if (parseInt(str.slice(3, 9), 10) === 0) return false;
```

Comment:
The original commit here intended to replace `substr` with `slice`, but here it also changes the length of the substring as well, If I am not completely mistaken.
I will do some tests locally and report back (maybe I am also just misunderstanding the way those ranges with substr and slice work)

Reply:
ok, just checked this and it seems that the syntax of the deprecated `substr` does indeed differ from slice.
so changes here are OK :-)